test/rules/typedef/parameter-arrow/test.ts.lint

```diff
@@ -0,0 +1,7 @@
+var NoTypesFn = function (a, b) {}
+
+var NoTypesArrowFn = (
```

Comment:
You can actually simplify this a little to improve readability here and below. (I know the test system isn't very well documented yet, sorry!)

```
var NoTypesArrowFn = (a, b) => {}
                       ~     [expected parameter-arrow: 'a' to have a typedef]
                          ~  [expected parameter-arrow: 'b' to have a typedef]
```